pkg/kubelet/config/flags.go

```diff
@@ -49,11 +47,34 @@ type ContainerRuntimeOptions struct {
 // AddFlags adds flags to the container runtime, according to ContainerRuntimeOptions.
 func (s *ContainerRuntimeOptions) AddFlags(fs *pflag.FlagSet) {
 	// General settings.
-	fs.StringVar(&s.RuntimeCgroups, "runtime-cgroups", s.RuntimeCgroups, "Optional absolute name of cgroups to create and run the runtime in.")
-	fs.StringVar(&s.PodSandboxImage, "pod-infra-container-image", s.PodSandboxImage, fmt.Sprintf("Specified image will not be pruned by the image garbage collector. CRI implementations have their own configuration to set this image."))
-	fs.MarkDeprecated("pod-infra-container-image", "will be removed in a future release. Image garbage collector will get sandbox image information from CRI.")
+	fs.StringVar(
+		&s.RuntimeCgroups,
+		"runtime-cgroups",
+		s.RuntimeCgroups,
+		"Optional absolute name of cgroups to create and run the runtime in.",
+	)
```

Comment:
As far as I can see this project mostly follows another code style. I'm not sure it makes sense to change it, at least not in one place.

```
kubernetes (master) $ git grep 'fs\..*Var(' | wc -l 
599
kubernetes (master) $ git grep 'fs\..*Var($' | wc -l 
8
```

Reply:
Reverted my changes.